app/controllers/serials_controller.rb

```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
+    @serials.extend(PaginationInfoDecorator)
+    @serials.extend(SerialsDecorator)
```

Comment:
I think `SerialsDecorator` depends on `PaginationInfoDecorator`. ( https://github.com/bm-sms/daimon-news/pull/551/files#diff-f7ddda9786b3de2ac334a4bc6f8d5751R5 👀 )
Can you include `PaginationInfoDecorator` into `SerialsDecorator`?